### PR TITLE
expand customer data

### PR DESCRIPTION
**Pull request by DerCalli**
**Description**

plugin expands transfer with customer-data via customer-facade

- move reference to customer-transfer
- fix typing error
- delete persistence-layer, replaced by customer-facade
- unit-tests
...

### CODE REVIEW

**Review by julianzimmermann — APPROVED** [review]

File: bundles/return-labels-rest-api-customer-connector/composer.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "fond-of-oryx/return-labels-rest-api-customer-connector",
+  "description": "Return Labels Rest Api Extension Module",
```
Comment (daniel-rose) [inline bundles/return-labels-rest-api-customer-connector/composer.json:3]:
```suggestion
  "description": "Return Labels Rest Api Customer Connector Module",
```

File: bundles/return-labels-rest-api-customer-connector/composer.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Pascal Fischer",
+      "email": "p-fischer@gmx.net"
+    }
+  ],
+  "require": {
+    "php": ">=7.3",
+    "spryker/customer": "^7.0.0"
```
Comment (daniel-rose) [inline bundles/return-labels-rest-api-customer-connector/composer.json:13]:
```suggestion
    "fond-of-oryx/return-labels-rest-api-extension": "^1.0.0",
    "spryker/customer": "^6.0.0 || ^7.0.0"
```

File: bundles/return-labels-rest-api-customer-connector/src/FondOfOryx/Zed/ReturnLabelsRestApiCustomerConnector/ReturnLabelsRestApiCustomerConnectorDependencyProvider.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+namespace FondOfOryx\Zed\ReturnLabelsRestApiCustomerConnector;
+
+use FondOfOryx\Zed\ReturnLabelsRestApiCustomerConnector\Dependency\Facade\ReturnLabelsRestApiCustomerConnectorToCustomerFacadeBridge;
+use Orm\Zed\Customer\Persistence\SpyCustomerQuery;
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+
+class ReturnLabelsRestApiCustomerConnectorDependencyProvider extends AbstractBundleDependencyProvider
+{
+    public const PROPEL_QUERY_CUSTOMER_QUERY = 'PROPEL_QUERY_CUSTOMER_QUERY';
```
Comment (daniel-rose) [inline bundles/return-labels-rest-api-customer-connector/src/FondOfOryx/Zed/ReturnLabelsRestApiCustomerConnector/ReturnLabelsRestApiCustomerConnectorDependencyProvider.php:12]: Do you need this?

File: composer.json [hunk 4]
```diff
@@ -1,283 +1,288 @@
...
+            "FondOfOryx\\Zed\\SplittableTotalsRestApi\\": "bundles/splittable-totals-rest-api/src/FondOfOryx/Zed/SplittableTotalsRestApi",
+            "FondOfOryx\\Zed\\SplittableTotalsRestApiCompanyUnitAddressConnector\\": "bundles/splittable-totals-rest-api-company-unit-address-connector/src/FondOfOryx/Zed/SplittableTotalsRestApiCompanyUnitAddressConnector",
+            "FondOfOryx\\Zed\\SplittableTotalsRestApiExtension\\": "bundles/splittable-totals-rest-api-extension/src/FondOfOryx/Zed/SplittableTotalsRestApiExtension",
+            "FondOfOryx\\Zed\\SplittableTotals\\": "bundles/splittable-totals/src/FondOfOryx/Zed/SplittableTotals",
+            "FondOfOryx\\Zed\\TaxCalculationConnector\\": "bundles/tax-calculation-connector/src/FondOfOryx/Zed/TaxCalculationConnector"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "FondOfOryx\\Client\\AvailabilityAlert\\": "bundles/availability-alert/src/FondOfOryx/Client/AvailabilityAlert",
```
Comment (daniel-rose) [inline composer.json:205]: src --> tests

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**